src/js/Pickers/ClockFace.js

```diff
@@ -35,6 +35,16 @@ export default class ClockFace extends PureComponent {
      * if the locale uses them.
      */
     timePeriod: PropTypes.string,
+
+    /**
+     * If true the hover mode of the Time Picker is activated.
+     * In hover mode no clicks are required to start selecting an hour
+     * and the timemode switches automatically when a time was chosen.
+     * When a minute is selected the chosen time is applied automatically.
+     */
+    hoverMode: PropTypes.bool,
+
+    onTimeChosen: PropTypes.func,
```

Comment:
Would you mind making this required and updating the ClockFace tests to include this prop? Or do an if-exists check before calling it?